src/SFA.DAS.ProviderRegistrations.Web/Views/Error/403.cshtml

```diff
@@ -1,12 +1,12 @@
 ﻿@using SFA.DAS.ProviderRegistrations.Web.Extensions
 @{
-    ViewData["Title"] = "Access denied";
+    ViewData["Title"] = "You do not have permission to access this page";
```

Comment:
If the theme is for the Title and the H1 to be the same value, could create a variable and set to it.

Reply:
Lets just keep as it is then